tests/cluster/schema_manager/test_schema_manager.py

```diff
@@ -25,16 +31,14 @@ def test_get_sql_schema(self):
         self.assertEqual(schema, expected_schema)
 
     def test_get_schema_as_string(self):
-        Configurator().add_resource_path(extras)
 
         schema = SchemaManager().get_schema_as_string(
             schema_identifier="SchemaTestTable2"
         )
 
-        self.assertEqual(schema, "a INTEGER, b STRING,")
+        self.assertEqual(schema, "a int, b string")
```

Comment:
@Okami1 You must not have run this before. The `simpleString()` method returns a lower-case short form like `int` and never `INTEGER`